src/index.js

```diff
@@ -11,7 +12,8 @@ ReactDOM.render((
     <Router>
         <Route exact path="/game" component={App} />
         <Route exact path="/styles" component={Styles} />
-        <Route exact path="/" component={App} />
+        <Route exact path="/pet-simulator" component={StartingPage} />
```

Comment:
Given the changes I made in `master` to fix our gh-pages routing issue, this Route should probably have the attribute `exact path="/"` instead of `exact path="/pet-simulator"`.

Reply:
I'll make the change. Thanks for the feedback!